src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -52,18 +70,24 @@ export class ChoiceGroup extends React.Component<IChoiceGroupProps, IChoiceGroup
     return (
       // Need to assign role application on containing div because JAWS doesnt call OnKeyDown without this role
       <div role='application' className={ className }>
-        <div className='ms-ChoiceFieldGroup' role='radiogroup'  aria-labelledby={ this.props.label ? this._id + '-label' : '' }>
+        <div
+          className='ms-ChoiceFieldGroup'
+          role='radiogroup'
```

Comment:
That's what we want: only 1 item to be checked at a time in this component. I think the name of the component 'ChoiceField' is confusing. I'll add role="radio" to its children.